src/Host/Client/Impl/Host/BrokerClient.cs

```diff
@@ -87,6 +84,12 @@ internal abstract class BrokerClient : IBrokerClient {
             }
         }
 
+        public async Task<AboutHost> GetHostInformationAsync(CancellationToken cancellationToken) {
+            var response = await HttpClient.GetAsync("/about", cancellationToken);
```

Comment:
Command that uses it will propagate exception to VS which typically handles and shows the error box.